types/luxon/src/info.d.ts

```diff
@@ -0,0 +1,174 @@
+import { CalendarSystem, NumberingSystem, StringUnitLength, UnitLength } from './misc';
+import { Zone } from './zone';
+
+export interface InfoOptions {
+    locale?: string;
+}
+
+export interface InfoUnitOptions extends InfoOptions {
+    numberingSystem?: NumberingSystem;
+}
+
+// @deprecated
```

Comment:
```suggestion
// deprecated
```
JSDoc won't work, let's be consistent now

Reply:
Ah If I change it to `/** @deprecated */` then my IDE will show interface usage as deprecated (strikethrough).

Reply:
Changed the comment type so the tag is recognized